src/main/java/mekanism/client/render/tileentity/RenderChemicalCrystallizer.java

```diff
@@ -16,7 +16,7 @@
 	private ModelChemicalCrystallizer model = new ModelChemicalCrystallizer();
 
 	@Override
-	public void renderTileEntityAt(TileEntityChemicalCrystallizer tileEntity, double x, double y, double z, float partialTick, int destroyStage)
+	public void func_192841_a(TileEntityChemicalCrystallizer tileEntity, double x, double y, double z, float partialTick, int destroyStage, float p_192841_10_)
```

Comment:
wut?? Update MCP mappings to something post-1.12 release, like `snapshot_20170615`